#! /bin/sh /usr/share/dpatch/dpatch-run
## gcc43.dpatch by Francesco Paolo Lovergine <frankie@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to be compliant with gcc 4.3

@DPATCH@
diff -urNad gdal-1.4.1~/frmts/idrisi/IdrisiDataset.cpp gdal-1.4.1/frmts/idrisi/IdrisiDataset.cpp
--- gdal-1.4.1~/frmts/idrisi/IdrisiDataset.cpp	2007-04-10 17:15:19.000000000 +0200
+++ gdal-1.4.1/frmts/idrisi/IdrisiDataset.cpp	2007-05-18 22:14:49.000000000 +0200
@@ -34,6 +34,7 @@
 #include "ogr_spatialref.h"
 #include "gdal_pam.h"
 #include "gdal_alg.h"
+#include <climits>
 
 CPL_CVSID("$Id: IdrisiDataset.cpp 10646 2007-01-18 02:38:10Z warmerdam $");
 
diff -urNad gdal-1.4.1~/frmts/ilwis/ilwisdataset.cpp gdal-1.4.1/frmts/ilwis/ilwisdataset.cpp
--- gdal-1.4.1~/frmts/ilwis/ilwisdataset.cpp	2007-04-10 17:15:31.000000000 +0200
+++ gdal-1.4.1/frmts/ilwis/ilwisdataset.cpp	2007-05-18 22:14:31.000000000 +0200
@@ -29,6 +29,7 @@
 
 #include "ilwisdataset.h"
 #include <float.h>
+#include <climits>
 
 // IniFile.cpp: implementation of the IniFile class.
 //
